src/main/java/com/conveyal/r5/analyst/cluster/WorkerMain.java

```diff
@@ -0,0 +1,29 @@
+package com.conveyal.r5.analyst.cluster;
+
+import com.conveyal.analysis.components.WorkerComponents;
+
+import java.util.Arrays;
+
+/**
+ * Old R5 JARs before December 2020 had main-class set to R5Main, which used the first command line parameter to
+ * select between starting an analysis worker and a point-to-point routing server. We no longer use the latter option,
+ * but we want to allow the backend to start all worker versions in exactly the same way, so we ignore the command line
+ * parameters. If we ever want to allow other roles, we can check that this parameter equals 'worker'.
+ */
+public class WorkerMain {
+
+    public static void main (String[] args) {
+        System.out.println("=== Conveyal R5 ===");
```

Comment:
_important_: loss of ASCII art?

Reply:
I do like ASCII art but I think I often saw it garbled in logs.